Unified/closor.py

```diff
@@ -540,10 +541,11 @@ def close(self):
                             to_DDM = True
 
                         ## by typical enabling
-                        if tier in UC.get("tiers_to_DDM"):
+                        tiers_to_skip = UC.get("tiers_to_skip_nonrelval")
```

Comment:
same comment here.

Reply:
Same answer as above.

Reply:
This one also looks okay to skip. However, if you want to keep the same logic, then I think an additional check needs to be implemented a few lines above, something like:
```
if tier in tiers_to_skip:
    pass
elif campaign and campaign in CI.campaigns and 'toDDM' in CI.campaigns[campaign] and tier in CI.campaigns[campaign]['toDDM']:
                            to_DDM = True
```
because we do not want to set `to_DDM=True` for tiers that cannot go to DDM.